sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClient.cs

```diff
@@ -629,7 +628,7 @@ public virtual AsyncPageable<ConfigurationSetting> GetConfigurationSettingsAsync
             var label = selector.LabelFilter;
 
             RequestContext context = CreateRequestContext(ErrorOptions.Default, cancellationToken);
-            IEnumerable<string> fieldsString = selector.Fields == SettingFields.All ? null : selector.Fields.ToString().ToLowerInvariant().Replace("isreadonly", "locked").Split(',');
+            IEnumerable<string> fieldsString = selector.Fields.Split();
```

Comment:
Bug happened because our logic was parsing "SettingFields.ContentType" into "contenttype", while the service expects "content_type". Not mentioned in the original bug, but the same issue can be observed with "LastModified" and "last_modified".